app/javascript/mastodon/components/status_quoted.tsx

```diff
@@ -0,0 +1,117 @@
+import { FormattedMessage } from 'react-intl';
+
+import classNames from 'classnames';
+
+import type { Map as ImmutableMap } from 'immutable';
+
+import { Icon } from 'mastodon/components/icon';
+import StatusContainer from 'mastodon/containers/status_container';
+import { useAppSelector } from 'mastodon/store';
+
+import QuoteIcon from '../../images/quote.svg?react';
+
+const QuoteWrapper: React.FC<{
+  isError?: boolean;
+  children: React.ReactNode;
```

Comment:
this doesn't make sense without a child as it's a wrapper. perhaps `ReactElement` or something is better?